nerfstudio/scripts/render.py

```diff
@@ -47,11 +47,19 @@
 from torch import Tensor
 from typing_extensions import Annotated
 
+from scipy.spatial.transform import Rotation as R
```

Comment:
Personally I'm not a fan of using `R` just in case someone uses that letter somewhere in the file. What do you think?

Reply:
Agreed I got rid of the `as R`